src/stores/Widgets/SpectralProfileWidget/SpectralProfileWidgetStore.ts

```diff
@@ -83,11 +83,22 @@ export class SpectralProfileWidgetStore extends RegionWidgetStore {
     readonly profileSelectionStore: SpectralProfileSelectionStore;
     readonly fittingStore: ProfileFittingStore;
 
+    /**
+     * Set region for generating moment maps.
+     *
+     * @param fileId - File ID number.
+     * @param regionId - Region ID number.
+     */
     @override setRegionId = (fileId: number, regionId: number) => {
```

Comment:
Perhaps this function is for the spectral profiler rather than moment maps. The description is the same as `selectMomentRegion`.